members/views.py

```diff
@@ -130,6 +133,88 @@ def members_bankcollection_list(request):
     else:
         return HttpResponseNotAllowed('you are not allowed to use this method')
 
+@login_required
+def members_bank(request):
+    if not request.user.is_superuser:
+        return HttpResponseNotAllowed('you are not allowed to use this method')
+    return render(request, 'members/member_bank.html')
+
+@login_required
+def members_bankcollection_sepa(request):
+    if not request.user.is_superuser:
+        return HttpResponseNotAllowed('you are not allowed to use this method')
+
+    if request.method != "POST":
+        return redirect("/member/bank/")
+
+    # get members that are active and have monthly collection activated
+    # 4 = monthly
+    members_to_collect_from = get_active_members()\
+        .filter(paymentinfo__bank_collection_allowed=True)\
+        .filter(paymentinfo__bank_collection_mode__id=4)
+
+    if len(members_to_collect_from) == 0:
+        return HttpResponse('Error: no members to collect from.')
+
+    sepa = SepaDD({
+        "name": settings.HOS_SEPA_CREDITOR_NAME,
+        "IBAN": settings.HOS_SEPA_CREDITOR_IBAN,
+        "BIC": settings.HOS_SEPA_CREDITOR_BIC,
+        "batch": settings.HOS_SEPA_BATCH,
+        "creditor_id": settings.HOS_SEPA_CREDITOR_ID,
+        "currency": settings.HOS_SEPA_CURRENCY,
+        "instrument": 'CORE'
+    }, schema=settings.HOS_SEPA_SCHEMA)
+
+    for member in members_to_collect_from:
+        debt = m.contactinfo.get_debt_for_month(date.today())
+        if debt > 0:
+            pmi = member.paymentinfo
+            # on the first debit initiation, set the mandate signing date
+            if not pmi.bank_account_date_of_signing:
+                pmi.bank_account_date_of_signing = date.today()
+                pmi.save()
+                payment_type = "FRST"
+                collection_date = date.today() + timedelta(days=+5)
+            else:
+                payment_type = "RCUR"
+                collection_date = date.today() + timedelta(days=+3)
```

Comment:
we should actually check and save if the mandate is used the first time but meh, since we don't actually know for sure if the export is uploaded there are worse problems.
We're refactoring this when we're introducing real collections.